### ISSUE TITLE
Sklearn deprecation warnings

### ISSUE THREAD

**Issue by scarere** [DIAGNijmegen/picai_eval#18]
Deprecation warnings appear anytime a metric is calculated/accessed via the Metrics object

![Screenshot from 2024-07-24 10-54-16](https://github.com/user-attachments/assets/24cf68e5-9831-41d4-a04d-b3cec066605e)

**Comment by joeranbosma** [comment]
Dear @scarere, thanks for raising this issue! It appears scikit-learn has updated its keyword in version 1.5, I'll update `picai_eval` to resolve this issue.

**Comment by joeranbosma** [comment]
The issue is resolved in this PR: https://github.com/DIAGNijmegen/picai_eval/pull/19
Once approved by the other maintainer of picai_eval, we will update this module on PyPI. Updating to version 1.4.7 will then resolve this deprecation warning.
